### DISCUSSION TITLE
Process could not be started [The system cannot find the path specified. ]

### DISCUSSION THREAD

**Discussion by VirendraArekar** [laravel/framework#38676] [Q&A]
MAIL_MAILER=smtp
MAIL_HOST=smtp.mailtrap.io
MAIL_PORT=2525
MAIL_USERNAME=############
MAIL_PASSWORD=############
MAIL_ENCRYPTION=tls
MAIL_FROM_ADDRESS=#############@gmail.com
MAIL_FROM_NAME="${APP_NAME}"

For sending mail it show me this error. Even I try it with different mail package but same error. But if try fresh Laravel installation it work for sending mail. I am using Laravel 7. Please help me.

**Comment by adityadees** [comment]
Hello, have you tried to clear your config and cache?
```
php artisan config:clear 
php artisan cache:clear 
```

And also check your host and port using `telnet host port`?
Example : `telnet smtp.mailtrap.io 2525`
I have tested your SMTP host and port and cannot connect

these are the results I'm using `telnet smtp.mailtrap.io 25`
![telnet](https://user-images.githubusercontent.com/37553901/132146077-5ab2c97a-b3c1-4d8f-9d58-f9f61d3c6100.png)

so you need to change
```
MAIL_MAILER=smtp
MAIL_HOST=smtp.mailtrap.io
MAIL_PORT=25
```

but if the port and host not working, You can try to use this 
```
SMTP
Host: mailtrap.io or smtp.mailtrap.io (MX)
Port: 25 or 465 or 2525
```

and don't forget, make sure you already clear cache before test it
